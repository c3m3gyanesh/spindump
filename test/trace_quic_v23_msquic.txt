This is a test of using the picoquic implementation against the publicly available Msquic server, using a regular connection and a retry. These commands were used:

    ./picoquic/picoquicdemo -l ./test.log -L -a "hq-23" -v ff000017 quic.westus.cloudapp.azure.com 4433 /5000
    ./picoquic/picoquicdemo -l ./test.log -L -a "hq-23" -v ff000017 quic.westus.cloudapp.azure.com 4434 /5000
