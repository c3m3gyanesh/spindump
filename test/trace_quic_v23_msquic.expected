QUIC 172.16.114.251 <-> 138.91.188.147 080e59f33f0849ac-4fc0129403ede438 (56571:4433) at 1571175338481030 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 138.91.188.147 080e59f33f0849ac-084f7438aa20cea2 (56571:4433) at 1571175338490705 measurement up right 9675 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 138.91.188.147 0e59f33f0849ac4b-084f7438aa20cea2 (56571:4433) at 1571175338498059 spinflip up 0-1 responder packets 3 2 bytes 1570 500 bandwidth 1570 500 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 138.91.188.147 0864a76d695423fa-0d1666bb2a5e8dc9 (58720:4434) at 1571175342094387 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 138.91.188.147 64a76d695423fac2-082aa802227b60b3 (58720:4434) at 1571175342117651 measurement starting left 1333 packets 1 1 bytes 1280 128 bandwidth 1280 128 note "V23,no spin"
QUIC 172.16.114.251 <-> 138.91.188.147 64a76d695423fac2-082a843fe317ac38 (58720:4434) at 1571175342127614 measurement up right 9963 packets 2 1 bytes 2560 128 bandwidth 2560 128 note "V23,no spin"
QUIC 172.16.114.251 <-> 138.91.188.147 64a76d695423fac2-082a843fe317ac38 (58720:4434) at 1571175342145650 spinflip up 0-1 initiator packets 6 6 bytes 5661 3670 bandwidth 5661 3670 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 138.91.188.147 64a76d695423fac2-082a843fe317ac38 (58720:4434) at 1571175342151163 spinflip up 0-1 responder packets 8 7 bytes 5809 3742 bandwidth 5809 3742 note "V23,spinning"
QUIC 172.16.114.251 <-> 138.91.188.147 64a76d695423fac2-082a843fe317ac38 (58720:4434) at 1571175342151163 measurement up right 5513 packets 8 7 bytes 5809 3742 bandwidth 5809 3742 note "V23,spinning"
